apps/prairielearn/package.json

```diff
@@ -68,7 +68,7 @@
     "body-parser": "^1.20.2",
     "bootstrap": "^4.6.2",
     "bootstrap-icons": "^1.11.3",
-    "bootstrap-table": "1.23.2",
+    "bootstrap-table": "1.21.2",
```

Comment:
Should this be added as a separate "group" in dependabot?

Reply:
IMO we should probably just add an [`ignore` condition](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore) for it.

Reply:
#10469